ecl/hql/hqlexpr.cpp

```diff
@@ -13046,6 +13058,10 @@ IHqlExpression * createExternalFuncdefFromInternal(IHqlExpression * funcdef)
         attrs.append(*LINK(cachedContextAttribute));
     if (functionBodyIsActivity(body))
         attrs.append(*createAttribute(activityAtom));
+    
+    IHqlExpression* passParamAttr = getFunctionBodyAttribute(body, passParameterMetaAtom);
+    if (passParamAttr != NULL && getBoolAttributeValue(passParamAttr))
```

Comment:
Minor improvement:  If getBoolAttributeValue() returned false when it was passed a null value, then this code, and the other places it is called could be simplified.